<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionL</class>
 <widget class="QMainWindow" name="GestionL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="gLTabs">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1181</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="gLTab1">
     <attribute name="title">
      <string>Locale info</string>
     </attribute>
     <widget class="QLabel" name="gLlabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>-10</y>
        <width>1171</width>
        <height>631</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Titillium Web</family>
        <pointsize>22</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/img/Untitled.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>22</pointsize>
        <bold>false</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Gestion de locaux</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-Adresse :</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-Responsable :</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>350</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-Numero t√©lephone :</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>420</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-Email :</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>ID Locale :</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_8">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>220</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-Heure d'ouverture :</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_9">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>320</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-Service disponible :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gLpushbutton_afficher">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>530</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>afficher Locaux</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gLpushbutton_modifier">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>530</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>181</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/output-onlinepngtools (1).png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_17">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>-20</y>
        <width>401</width>
        <height>451</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/img/461764196_1062489365880787_1649872201096511266_n-removebg-preview.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_19">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>360</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/img/461764196_1062489365880787_1649872201096511266_n-removebg-preview.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="gLlineEdit_idlocal">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gLlineEdit_adresse">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gLlineEdit_Responsable">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>290</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gLlineEdit_Num_telephone">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>360</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gLlineEdit_heure">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>230</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gLlineEdit_email">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>420</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gLpushbutton_supprimer">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>528</y>
        <width>83</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gLpushbutton_ajouter">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>530</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gLpushButton_aff">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>130</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QComboBox" name="gLcomboBox_Service">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>330</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images.qrc">:/img/output-onlinepngtools (1).png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="gLpushButton_showGraph">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>480</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/img.icons8.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_10">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>530</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>statistique</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gLlineEdit_temperature">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>130</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="gLpushButton_updateTemperature">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>130</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>update ¬∞C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gLpushButton_note">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>480</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/notebook.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="gLlabel_11">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>530</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Notes</string>
      </property>
     </widget>
     <zorder>gLlabel</zorder>
     <zorder>gLlabel_17</zorder>
     <zorder>gLlabel_19</zorder>
     <zorder>gLlabel_2</zorder>
     <zorder>gLlabel_3</zorder>
     <zorder>gLlabel_4</zorder>
     <zorder>gLlabel_5</zorder>
     <zorder>gLlabel_6</zorder>
     <zorder>gLlabel_7</zorder>
     <zorder>gLlabel_8</zorder>
     <zorder>gLlabel_9</zorder>
     <zorder>gLpushbutton_afficher</zorder>
     <zorder>gLpushbutton_modifier</zorder>
     <zorder>gLlabel_14</zorder>
     <zorder>gLlineEdit_idlocal</zorder>
     <zorder>gLlineEdit_adresse</zorder>
     <zorder>gLlineEdit_Responsable</zorder>
     <zorder>gLlineEdit_Num_telephone</zorder>
     <zorder>gLlineEdit_heure</zorder>
     <zorder>gLlineEdit_email</zorder>
     <zorder>gLpushbutton_supprimer</zorder>
     <zorder>gLpushbutton_ajouter</zorder>
     <zorder>gLpushButton_aff</zorder>
     <zorder>gLcomboBox_Service</zorder>
     <zorder>gLlabel_22</zorder>
     <zorder>gLpushButton_showGraph</zorder>
     <zorder>gLlabel_10</zorder>
     <zorder>gLlineEdit_temperature</zorder>
     <zorder>gLpushButton_updateTemperature</zorder>
     <zorder>gLpushButton_note</zorder>
     <zorder>gLlabel_11</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="gLstatusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
